src/components/Home/Home.jsx

```diff
@@ -1,12 +1,17 @@
 import './home.css';
-import {
-    LowMaintainanceCategory,
-    IndoorCategory,
-    AirPurifierCategory,
-    HeroImage
-  } from "../../assets/index";
+import {useEffect, useState} from "react";
+import axios from 'axios';
+import {HeroImage} from "../../assets/index";
 
 export const Home = () => {
+const [categories, setCateogries] = useState([]);
+useEffect(() => {
+    (async() => {
+      const {data: {categories}} = await axios.get('/api/categories');
```

Comment:
suggestion: you can make custom hook of axios and name it something like "useAxios" ,so whenever you want to use axios you can access it through useAxios and then you dont need to call axios everywhere.